AltServer/AltServerApp.cpp

```diff
@@ -326,6 +327,7 @@ AltServerApp* AltServerApp::instance()
 
 AltServerApp::AltServerApp() : _appGroupSemaphore(1)
 {
+	CLSIDFromString(L"{96A5974D-D3A2-909A-B6BD-4FF84E7880F7}", &_notificationIconGUID);

```

Comment:
Unlike the official version of AltServer which generates a random GUID each time, a fixed one is used here. 
It may not be appropriate to submit this modification together in this PR, which was caused by a mistake of mine.